test/unit/url_view.cpp

```diff
@@ -56,6 +57,14 @@ class url_view_test
             BOOST_TEST_EQ(u1.data(), u1.data());
         }
 
+        // operator=(url_view_base const&)
+        {
+            url u1("x://y/z?#");
+            url_view u2 = u1;
+            u2 = u1;
+            BOOST_TEST_EQ(u1.data(), u1.data());
```

Comment:
I thought I already did this test??

Reply:
if you are going to have more than one test per member please group them together under one comment, like this:
```
        // operator=(url_view const&)
        {
            url_view u1("x://y/z?#");
            url_view u2;
            u2 = u1;
            BOOST_TEST_EQ(u2.data(), u1.data());
        }
        {
            // self-assignment
            url_view u1("x://y/z?#");
            u1 = u1;
            BOOST_TEST_EQ(u1.data(), u1.data());
        }
```

and add a comment explaining what the difference is